cpp/pmft/PMFTXYZ.cc

```diff
@@ -78,7 +76,7 @@ void PMFTXYZ::accumulate(const locality::NeighborQuery* neighbor_query,
         for (unsigned int k = 0; k < n_faces; k++)
         {
             // create the extra quaternion
-            quat<float> qe(face_orientations[q_i(k, neighbor_bond.ref_id)]);
+            quat<float> qe(face_orientations[util::ManagedArray<unsigned int>::getIndex({neighbor_query->getNPoints(), n_faces}, {neighbor_bond.ref_id, k})]);
```

Comment:
What this does is not clear to me at all. I would prefer to make this into two lines or add comments or something like that.

Reply:
Alternatively don't make the indexing method `static`, and make it aware of the shape of the underlying array?

Reply:
I'm not exactly sure what you're suggesting here. In the interest of moving forward, I made an issue to track this and I'm just going to resolve this and merge.